packages/Search/src/details/DTK_DetailsTreeConstruction_def.hpp

```diff
@@ -185,8 +187,10 @@ class CalculateBoundingBoxesFunctor
             if ( Kokkos::atomic_compare_exchange_strong(
                      &_flags( node - _root ), 0, 1 ) )
                 break;
-            for ( Node *child : {node->children.first, node->children.second} )
-                expand( node->bounding_box, child->bounding_box );
+
+            node->bounding_box = node->children.first->bounding_box;
```

Comment:
Add a comment here that `node->bounding_box` is not initialized before this code.